include/CommandHandler/src/CommandHandler.cpp

```diff
@@ -302,20 +307,21 @@ void CommandHandler::HandleVrfy(SocketWrapper& socket_wrapper, const std::string
             Logger::LogWarning("User does not exist: " + user_name);
             socket_wrapper.SendResponseAsync("550 User does not exist\r\n").get();
         }
-    } catch (const MailException& e) {
+    }
+    catch (const MailException& e)
+    {
         Logger::LogError("MailException caught in HandleVrfy: " + std::string(e.what()));
         ErrorHandler::HandleError("Handle VRFY", e, socket_wrapper, "550 Internal Server Error\r\n");
     }
     Logger::LogDebug("Exiting CommandHandler::HandleVrfy");
 }
 
-
 void CommandHandler::HandleExpn(SocketWrapper& socket_wrapper, const std::string& line)
 {
     Logger::LogDebug("Entering CommandHandler::HandleExpn");
     Logger::LogTrace("CommandHandler::HandleExpn parameter: SocketWrapper reference");
 
-    const std::string mailing_list = line.substr(5);
+    const std::string mailing_list = line.substr(MAILING_LIST_PREFIX_LENGTH);
     Logger::LogTrace("EXPN command received for mailing list: " + mailing_list);
 
```

Comment:
Should add checking if ```mailing_list``` is empty, maybe, you need to check if  ```mailing_list```  is syntactically valid